docs/pages/reference/backends.mdx

```diff
@@ -556,6 +560,9 @@ teleport:
     write_min_capacity: int
     write_max_capacity: int
     write_target_value: float
+    # Enables On-Demand billing
```

Comment:
These settings only take effect if the table does not exist and Teleport has to create it, right?

If so, that's not clear and we should probably clarify.